{"name": "测试数据：【 wintest，123456，True，0，登录成功】", "status": "failed", "statusDetails": {"message": "AssertionError: 接口返回码是 【 500 】, 返回信息：JSON parse error: Unrecognized token 'username': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false'); nested exception is com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'username': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')\n   at [Source: (org.springframework.util.StreamUtils$NonClosingInputStream); line: 1, column: 10] \nassert False == True\n +  where False = <core.result_base.ResultBase object at 0x10b496130>.success", "trace": "self = <testcases.api_test.test_03_login.TestUserLogin object at 0x10b45a760>, username = 'wintest', password = '123456', except_result = True, except_code = 0, except_msg = '登录成功'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = login_user(username, password)\n        step_1(username)\n    \n>       assert result.success == except_result, result.error\nE       AssertionError: 接口返回码是 【 500 】, 返回信息：JSON parse error: Unrecognized token 'username': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false'); nested exception is com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'username': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')\nE          at [Source: (org.springframework.util.StreamUtils$NonClosingInputStream); line: 1, column: 10] \nE       assert False == True\nE        +  where False = <core.result_base.ResultBase object at 0x10b496130>.success\n\ntestcases/api_test/test_03_login.py:31: AssertionError"}, "description": "该用例是针对获取用户登录接口的测试", "steps": [{"name": "步骤1 ==>> 登录用户", "status": "passed", "parameters": [{"name": "username", "value": "'wintest'"}], "start": 1735908717296, "stop": 1735908717296}], "attachments": [{"name": "log", "source": "1701d7a5-39c8-4ad1-9aaa-30b265ca1b88-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "3eec7cd4-bcc1-4a36-b85f-da30d72001fd-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'wintest'"}, {"name": "password", "value": "'123456'"}, {"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "'登录成功'"}], "start": 1735908717202, "stop": 1735908717296, "uuid": "e0b2dbf4-19ba-418f-91ad-6eaa96f8c419", "historyId": "4ab02ebad4baabe28d2fb7352ad2343a", "testCaseId": "7272db244dabef390096479ae3196642", "fullName": "testcases.api_test.test_03_login.TestUserLogin#test_login_user", "labels": [{"name": "story", "value": "用例--登录用户"}, {"name": "severity", "value": "normal"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "feature", "value": "用户登录模块"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_03_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "zhouqideMBP.lan"}, {"name": "thread", "value": "4802-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_03_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}